treeherder/changelog/collector.py

```diff
@@ -0,0 +1,96 @@
+""" Collector, grabs changes in various sources and put them in a DB.
+"""
+import github3
+import json
+import os
+
+from treeherder.changelog import models
+from treeherder.changelog.filters import filter_out
+
+
+MAX_ITEMS = 100
+CFG = os.path.join(os.path.dirname(__file__), "repositories.json")
+
+with open(CFG) as f:
+    CFG = json.loads(f.read())
+
+
+class GitHub:
+    def __init__(self):
+        # XXX plug into Django-whatever-secrets-things
+        self.token = os.environ["GITHUB_TOKEN"]
+        self.id = os.environ["GITHUB_ID"]
```

Comment:
You can use `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET` environment variables.